models/stg_hubspot__owner.sql

```diff
@@ -1,4 +1,4 @@
-{{ config(enabled=fivetran_utils.enabled_vars(['hubspot_sales_enabled','hubspot_owner_enabled'])) }}
+{{ config(enabled=var('hubspot_owner_enabled', true)) }}
```

Comment:
Same comment as on the source -- we should probably note that this is no longer dependent on `hubspot_sales_enabled`